project_hr/README.rst

```diff
@@ -53,13 +53,13 @@ Configuration
 
 For setting employee categories at project level:
 
-#. Go to *Project > Dashboard*.
-#. Click on the 3 vertical dots for unfolding options and select "Settings".
+#. Go to *Project > Configuration > Settings*.
```

Comment:
This is not correct, and previous steps were correct. The only changed thing is that instead of `Dashboard`, main menu is now called `Projects` (*Project > Projects* in summary)